src/SFA.DAS.EmployerIncentives.Domain/ValueObjects/CollectionPeriod.cs

```diff
@@ -30,12 +31,18 @@ public CollectionPeriod(byte periodNumber, short academicYear)
         public DateTime CensusDate { get; }
         public short AcademicYear { get; }
         public bool Active { get; private set; }
+        public bool PeriodEndInProgress { get; private set; }
```

Comment:
The stories refer to Month End Processing.  Shouldn't this be named similarly?  e.g. MonthEndProcessingInProgress ?

Reply:
I'm tempted to just call it IsInProgress?